File: php/main4.php
<?php
function dangerous($v1,$v2){ return intdiv($v1,$v2); }
function foo($array,$counter){
  if($counter===0){ return dangerous($array[0], $counter); }
  return foo($array, $counter-1);
}
$array = array_fill(0, 1000, 0);
echo "The result is ".foo($array, 6).PHP_EOL;

File: swift/main4.swift
func dangerous(_ v1:Int, _ v2:Int) -> Int { v1 / v2 }
func foo(_ array:[Int], _ counter:Int) -> Int {
  if counter == 0 { return dangerous(array[0], counter) }
  return foo(array, counter-1)
}
let array = Array(repeating: 0, count: 1000)
print("The result is \(foo(array, 6))")

